packages/ripple-nuxt-tide/lib/core/tide-helper.js

```diff
@@ -45,7 +45,11 @@ export const metatagConverter = (metatagNormalized) => {
   metatagNormalized.forEach(element => {
     switch (element.tag) {
       case 'meta':
-        metatags[element.attributes.name] = element.attributes.content
+        if (element.attributes['http-equiv']) {
+          metatags.lang = element.attributes.content
```

Comment:
Hey @dylankelly this `http-equiv` can be something else like below. So we need a condition to check if it's `content-language`.
![Screen Shot 2020-09-09 at 5 22 15 pm](https://user-images.githubusercontent.com/31641325/92567442-0da8d600-f2c1-11ea-98b1-c3e963fda39b.png)

You can leave it to me, I will add this one and do a QA. Cheers.